data-source/jsonnet/england-wales/household/blocks/who-lives-here/anyone_usually_live_at.jsonnet

```diff
@@ -23,6 +23,7 @@ function(census_date) {
       when: [rules.hasPrimary],
     },
   ],
+  hide_on_section_summary: true,
```

Comment:
Any reason we mix case, in the schema its capitalised or is this a jsonnet thing?

Reply:
Can you be more specific? Do you mean the case of the boolean value?

Reply:
Yep

Reply:
JSON standard is booleans as `true` or `false` - same as JS :+1: